app/views/error/unauthorized.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="alert alert-danger">
+  <%= message %>
```

Comment:
now for the fun part... just call presenter for everything.. `presenter.message`